erpnext/selling/doctype/sales_order/sales_order.py

```
@@ -1665,10 +1682,7 @@ def update_item_for_packed_item(source, target, source_parent):
 						"margin_rate_or_amount",
 					],
 					"postprocess": update_item,
-					"condition": lambda doc: doc.ordered_qty < doc.stock_qty
-					and doc.supplier == supplier
-					and doc.item_code in items_to_map
-					and not is_product_bundle(doc.item_code),
+					"condition": lambda doc: filter_items(doc, supplier),
```

Ruff (0.14.5)

Function definition does not bind loop variable `supplier`

(B023)